frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -103,6 +103,8 @@ type AudioAnnotatorState = {
   taskStartTime: number,
   annotations: Array<Annotation>,
   currentDefaultTagAnnotation: string,
+  inAModal: boolean,
+  alphanumeric_keys: Array<string>,
```

Comment:
Why are `alphanumeric_keys` in state? It could just be a private member of the class component